config-vm.md

```diff
@@ -0,0 +1,49 @@
+# <a name="VirtualMachineSpecificContainerConfiguration" /> Virtual-machine-specific Container Configuration
+
+Virtual-machine-based containers require additional configuration to that specified in the [base spec configuration](config.md).
+
+This **optional** configuration is specified in a "VM" object:
+
+* **`imagePath`** (string, REQUIRED) host path to file that represents the root filesystem for the container virtual machine.
+* **`kernel`** (object, REQUIRED) specifies details of the kernel to boot the container virtual machine with.
+* **`hypervisor`** (object, REQUIRED) specifies details of the hypervisor that manages the container virtual machine.
```

Comment:
Do all VM-based runtimes support multiple hypervisors?  I'd expect this to be OPTIONAL, with an implementation-defined default.